style.css

```diff
@@ -0,0 +1,199 @@
+.form{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50% -50%);
+    width: 30%;
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+    padding: 2rem;
+    border-radius: 15px;
+    color: rgb(143, 140, 140);
+}
+*{
+    margin: 0px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+div{
+    display: block;
+}
+.head {
+    font-size: 1.5rem;
+    font-weight: 400;
+    text-align: center;
+    padding: 0.2rem 0;
+    color: #0e71ea;
+}
+ul {
+    display: block;
+    list-style-type: disc;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    padding-inline-start: 40px;
+}
+.form-content, ul {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-top: 1rem;
+}
+* {
+    padding: 0;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
```

Comment:
Once this is added on top of the CSS file, no need to repeat itself. Also, this can override the previous styles. The CSS file is interpreted by the browser from left to right and top to bottom. Take this example:
```CSS
.form {
margin: 20px
}

p {
color: red;
}
```

The above code will be overridden with this code:
```CSS
.form {
margin: 30%;
}

p {
color: blue;
}
```